app/assets/javascripts/app/views/help_view.js

```diff
@@ -133,6 +136,32 @@ app.views.Help = app.views.StaticContentView.extend({
     this.$('#faq').append(help_section.render().el);
   },
 
+  /**
+   * Returns the jQuery <a> element whose data-section property equals the given query
```

Comment:
When adding docs, let's do it right. The function does not just return any matching `<a>` on the page, it also has to have the class `faq-link`. How about saying "section title" or so?

I'm also not sure about the functions name, `findSection` or something like that seems more appropriate, a function that's executed for its return value rather than its side-effect should describe what it returns, not what it does.